Empire/Empire.h

```diff
@@ -181,13 +177,13 @@ struct FO_COMMON_API ProductionQueue {
         int             blocksize;                  ///< size of block to produce (default=1)
         int             remaining;                  ///< how many left to produce
         int             location;                   ///< the ID of the UniverseObject at which this item is being produced
-        float           allocated_pp;               ///< PP allocated to this ProductionQueue Element by Empire production update
-        float           progress;                   ///< fraction of this item that is complete.
-        float           progress_memory;            ///< updated by server turn processing; aides in allowing blocksize changes to be undone in same turn w/o progress loss
-        int             blocksize_memory;           ///< used along with progress_memory
-        int             turns_left_to_next_item;
-        int             turns_left_to_completion;
-        int             rally_point_id;
+        float           allocated_pp = 0.0f;        ///< PP allocated to this ProductionQueue Element by Empire production update
+        float           progress = 0.0f;            ///< fraction of this item that is complete.
+        float           progress_memory = 0.0f;     ///< updated by server turn processing; aides in allowing blocksize changes to be undone in same turn w/o progress loss
+        int             blocksize_memory = 1;       ///< used along with progress_memory
+        int             turns_left_to_next_item = -1;
+        int             turns_left_to_completion = -1;
+        int             rally_point_id = INVALID_OBJECT_ID;
         bool            paused;
```

Comment:
I was looking for (easy) cases where I would obviously save some lines of code. In this case, I only noticed the one line in the Empire.cpp file, and didn't notice the 3 ctor definitions in the .h file that have default parameters for "paused".

If it it okay, I'll save that for a future PR-- check how ProductionQueue::Element is being used in the code, and see if refactoring makes sense.